### PR TITLE
fixed external OAuth documentation code snippet comments

### PR DESCRIPTION
**Pull request by lghiur**

### CODE REVIEW

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md [hunk 1]
```diff
@@ -33,17 +33,14 @@ externalOAuthServer:
...
     introspection: # array for introspection details
       enabled: true/false
       clientID: # for introspection request
       clientSecret: # for introspection request, if empty will use oAuth.secret
       url: # token introspection endpoint
-      cache: #Possible NOT MVP #if enabled or timeout is 0 by default will cache by the time of JWT exp
+      cache: # Tyk caches the introspection response when enabled is set true inside the cache configuration of introspection
         enabled: true/false,
-        timeout: ... #optional field
+        timeout: 60 # Tyk caches the introspection result until the timeout runs out. The value is in seconds
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]: is 60 the default?

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md [hunk 2]
```diff
@@ -33,17 +33,14 @@ externalOAuthServer:
...
-      claims: #NOT MVP # should we use one single schema for all the claims?
-      - name: "iss"
-        schema: {JSON_schema}
     introspection: # array for introspection details
       enabled: true/false
       clientID: # for introspection request
       clientSecret: # for introspection request, if empty will use oAuth.secret
       url: # token introspection endpoint
-      cache: #Possible NOT MVP #if enabled or timeout is 0 by default will cache by the time of JWT exp
+      cache: # Tyk caches the introspection response when enabled is set true inside the cache configuration of introspection
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]:
```suggestion
      cache: # Tyk will cache the introspection response when `cache.enabled` is set to `true`
```

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md [hunk 3]
```diff
@@ -33,17 +33,14 @@ externalOAuthServer:
...
         enabled: true/false,
-        timeout: ... #optional field
+        timeout: 0 # Tyk caches the introspection result until the timeout runs out. The value is in seconds
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]:
```suggestion
        timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)
```

**Review by andyo-tyk — CHANGES_REQUESTED** [review]

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md (hunk 3 above)
Comment (letzya) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]: what does it mean zero timeout if the cache is enabled?

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md [hunk 4]
```diff
@@ -33,17 +33,14 @@ externalOAuthServer:
...
     introspection: # array for introspection details
       enabled: true/false
       clientID: # for introspection request
       clientSecret: # for introspection request, if empty will use oAuth.secret
       url: # token introspection endpoint
-      cache: #Possible NOT MVP #if enabled or timeout is 0 by default will cache by the time of JWT exp
+      cache: # Tyk will cache the introspection response when `cache.enabled` is set to `true`
         enabled: true/false,
-        timeout: ... #optional field
+        timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds)
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]: what does it mean zero timeout with cache enabled?

Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]: I wondered that! @lghiur?

Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]: It means that it is cached forever

Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]:
```suggestion
        timeout: 0 # Length of time for which Tyk should cache the introspection result; if set to zero (default) then cache does not expire (seconds)
```

**Review by andyo-tyk — DISMISSED** [review]

File: tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md [hunk 5]
```diff
@@ -33,17 +33,14 @@ externalOAuthServer:
...
     introspection: # array for introspection details
       enabled: true/false
       clientID: # for introspection request
       clientSecret: # for introspection request, if empty will use oAuth.secret
       url: # token introspection endpoint
-      cache: #Possible NOT MVP #if enabled or timeout is 0 by default will cache by the time of JWT exp
+      cache: # Tyk will cache the introspection response when `cache.enabled` is set to `true`
         enabled: true/false,
-        timeout: ... #optional field
+        timeout: 0 # Length of time for which Tyk should cache the introspection result (seconds). "0" means that the introspection result is cached by the end of life of the token
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/authentication-authorization/ext-oauth-middleware.md]:
```suggestion
        timeout: 0 # The duration (in seconds) for which Tyk will retain the introspection outcome in its cache. If the value is "0", it indicates that the introspection outcome will be stored in the cache until the token's expiration.
```

**Review by letzya — APPROVED** [review]

**Pull request merged**